Common/Defs/Misc/ShipDefs/Mechanoid/MysteriousArchotechSphere.xml

```diff
@@ -15,7 +15,6 @@
 				<key>?</key>
 				<value>
 					<shapeOrDef>ShipHullTileMech</shapeOrDef>
```

Comment:
this value tag is no longer closing, is this correct?

Reply:
Probably not, I'm trying to find some time to reimplement this into the newer code base, but I'm really busy at work these days.